packages/valory/skills/abstract_round_abci/test_tools/abci_app.py

```diff
@@ -62,37 +62,37 @@ def end_block(self) -> Tuple[MagicMock, MagicMock]:
 class ConcreteRoundB(_ConcreteRound):
     """Dummy instantiation of the AbstractRound class."""
 
-    allowed_tx_type = "payload_b"
+    payload_class = BaseTxPayload
```

Comment:
It's in principle ok to reuse payloads in roundassignments (see e.g. registration round). But for these test tools specifically it might lead to unintended side-effects. Please check

However, we might want to say you cannot ever reuse payloads and that there needs to be a 1:1 correspondence between payloads and rounds. It certainly makes the system much easier to argue about. And for the developer they simply have to add two line of class definition:
```
@dataclass
class New(Base):
```

Reply:
We should open an issue for that as it's beyond the scope here

Reply:
https://github.com/valory-xyz/open-autonomy/issues/1742